x/precisebank/types/constants.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+// IntegerCoinDenom is the denomination for integer coins that are managed by
+// x/bank. This is the "true" denomination of the coin, and is also used for
+// the reserve to back all fractional coins.
+const IntegerCoinDenom = "ukava"
```

Comment:
Filed https://github.com/Kava-Labs/kava/issues/1914, I'm still split if adding the staking and evm keeper params as depedencies is worth it.